specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -12241,10 +12272,37 @@
           "format": "date-time",
           "description": "The UTC timestamp when the operation began."
         },
-        "startedBy": {
+        "error": {
+          "$ref": "#/definitions/ApiError",
+          "readOnly": true,
+          "description": "The errors that were encountered during execution of the operation. The details array contains the list of them."
+        }
+      },
+      "description": "The Hotpatch Status for the virtual machine."
+    },
+    "HotpatchStatus": {
+      "properties": {
+        "hotpatchEnablementStatus": {
           "type": "string",
           "readOnly": true,
-          "description": "The person or system account that started the operation"
+          "description": "Represents what state of hotpatching opt in the VM is in. .<br /><br /> Possible values are:<br /><br /> **Unknown** - Safe default until initialized. <br /><br /> **PendingEvaluation** - Value reported when there's no data from the extension to work with. <br /><br /> **NotRequested** - Extension reports that there was no instruction to enable hotpatching. <br /><br /> **ActionRequired** - Extension reports that customer action is required to complete opt in. <br /><br /> **Enabled** - Extension reports that hotpatching enablement has completed.",
```

Comment:
Please use this instead for the description: 
"Represents the enablement status of hotpatching on the VM. .<br /><br /> Possible values are:<br /><br /> **Unknown** - Unknown status. <br /><br /> **PendingEvaluation** - Enablement status evaluation is pending on the VM.<br /><br /> **Disabled** - Hotpatching is disabled. <br /><br /> **ActionRequired** - Customer action is required to complete enablement. Further details will be in the error object.<br /><br /> **Enabled** - Hotpatching is enabled."

`"Represents the enablement status of hotpatching on the VM. .<br /><br /> Possible values are:<br /><br /> **Unknown** - Unknown status. <br /><br /> **PendingEvaluation** - Enablement status evaluation is pending on the VM.<br /><br /> **Disabled** - Hotpatching is disabled. <br /><br /> **ActionRequired** - Customer action is required to complete enablement. Further details will be in the error object.<br /><br /> **Enabled** - Hotpatching is enabled."`